### ISSUE TITLE
STIG is modifying /etc/fstab in such a way that causes the system to not fully boot afterwords.

### ISSUE THREAD

**Issue by atlas1Mc** [RedHatOfficial/ansible-role-rhel9-stig#8]
After running the RHEL9 STIG role the /etc/fstab is modified like so.
From:
UUID=8b4a4ce6-cc11-42e3-afc4-bbb13f950d40       /              xfs     defaults        0       0
UUID=af92dd76-6f1f-47ed-a0b7-126b19b67bd4       /boot          xfs     defaults,nodev,nosuid   0       0
UUID=7B77-95E7                                  /boot/efi      vfat    defaults,uid=0,gid=0,umask=077,shortname=winnt,nosuid,nodev     0       2
UUID=123fbbee-b02f-4428-9d94-cb270dd87686       /var           xfs     rw,seclabel,relatime,inode64,logbufs=8,logbsize=32k,noquota,nodev,nosuid 0 0
UUID=ca988382-8e79-4586-9105-e3d8a5611d2f       /home          xfs     rw,nodev,relatime,noexec,nosuid 0 0
UUID=5d8be316-3893-4446-b107-e52ba4c36356       /opt           xfs     rw,seclabel,relatime,inode64,logbufs=8,logbsize=32k,noquota,nodev 0 0
UUID=a091392f-e0cc-4f2c-ade3-2392fea6e5d1       /var/log       xfs     rw,seclabel,relatime,inode64,logbufs=8,logbsize=32k,noquota,nodev,nosuid,noexec 0 0
UUID=30894ca8-d5bf-4a11-b988-2a6cee959c74       /var/log/audit xfs     rw,seclabel,relatime,inode64,logbufs=8,logbsize=32k,noquota,nodev,nosuid,noexec 0 0
UUID=a6ee9272-3eb8-41a5-85c0-415055ba7621       /tmp           xfs     rw,seclabel,relatime,inode64,logbufs=8,logbsize=32k,noquota,nodev,noexec,nosuid 0 0
UUID=49c61de6-d0f8-4b95-80a0-0baa43908ab7       /var/tmp       xfs     rw,seclabel,relatime,inode64,logbufs=8,logbsize=32k,noquota,nodev,noexec,nosuid 0 0

To:
UUID=8b4a4ce6-cc11-42e3-afc4-bbb13f950d40       /              xfs     defaults        0       0
/dev/xvda3                                      /boot          xfs     rw,seclabel,relatime,attr2,inode64,logbufs=8,logbsize=32k,noquota,nodev 0 0
/dev/xvda2                                      /boot/efi      vfat    rw,relatime,fmask=0077,dmask=0077,codepage=437,iocharset=ascii,shortname=winnt,errors=remount-ro,nodev 0 0
UUID=123fbbee-b02f-4428-9d94-cb270dd87686       /var           xfs     rw,seclabel,relatime,inode64,logbufs=8,logbsize=32k,noquota,nodev,nosuid 0 0
UUID=ca988382-8e79-4586-9105-e3d8a5611d2f       /home          xfs     rw,nodev,relatime,noexec,nosuid 0 0
UUID=5d8be316-3893-4446-b107-e52ba4c36356       /opt           xfs     rw,seclabel,relatime,inode64,logbufs=8,logbsize=32k,noquota,nodev 0 0
UUID=a091392f-e0cc-4f2c-ade3-2392fea6e5d1       /var/log       xfs     rw,seclabel,relatime,inode64,logbufs=8,logbsize=32k,noquota,nodev,nosuid,noexec 0 0
UUID=30894ca8-d5bf-4a11-b988-2a6cee959c74       /var/log/audit xfs     rw,seclabel,relatime,inode64,logbufs=8,logbsize=32k,noquota,nodev,nosuid,noexec 0 0
UUID=a6ee9272-3eb8-41a5-85c0-415055ba7621       /tmp           xfs     rw,seclabel,relatime,inode64,logbufs=8,logbsize=32k,noquota,nodev,noexec,nosuid 0 0
UUID=49c61de6-d0f8-4b95-80a0-0baa43908ab7       /var/tmp       xfs     rw,seclabel,relatime,inode64,logbufs=8,logbsize=32k,noquota,nodev,noexec,nosuid 0 0
tmpfs                                           /dev/shm       tmpfs   rw,nosuid,nodev,noexec,seclabel,inode64 0 0
/dev/xvda2                                      /efi           vfat    rw,relatime,fmask=0077,dmask=0077,codepage=437,iocharset=ascii,shortname=winnt,errors=remount-ro,nodev 0 0

In summary the modification replaces the UUID for /boot, /boot/efi, and /efi to something /dev/xvda# version.

After this change and the server is rebooted the following error is seen in the console:
`Cannot open access to console, the root account is locked`

The build process includes taking the latest AWS RHEL9 build and using Packer and the Ansible RHEL9 role to STIG the image and create a new AMI.  When deployed the server is not able to fully boot.  

The fix that works:
Disable all the 26 rules in the "Restrict Partition Mount Options Group" in the STIG then the AMI boots just fine afterwords.

**Comment by jseiser** [comment]
@atlas1Mc 

I think you need to report this here: https://github.com/ComplianceAsCode/content

Its where the code is actually generated from.

**Comment by dmc5179** [comment]
@jseiser is correct. I need to disable issues on these repos and direct people to the complianceascode repos.

**Comment by dmc5179** [comment]
Issues should be reported here: https://github.com/ComplianceAsCode/content
